lib/ui/geometry.dart

```diff
@@ -122,37 +120,37 @@ abstract class OffsetBase {
 class Offset extends OffsetBase {
   /// Creates an offset. The first argument sets [dx], the horizontal component,
   /// and the second sets [dy], the vertical component.
-  const Offset(double dx, double dy) : super(dx, dy);
+  const Offset(double/*!*/ dx, double/*!*/ dy) : super(dx, dy);
```

Comment:
Optionally, add non-null asserts for dx/dy here for (hopefully very limited-time) future proofing.

Reply:
The `super` already does that (as of this PR).